test/common.js

```diff
@@ -354,3 +354,59 @@ exports.fileExists = function(pathname) {
     return false;
   }
 };
+
+
+
+(function() {
```

Comment:
Can I suggest moving this to the regression test file?  It's only used in one place and while I understand that the hard-coded timeouts can't be helped, I don't want to encourage new contributors to start using it in their own tests.